### ISSUE TITLE
Loading screen is bugged + having issue on the editing roles menu

### ISSUE THREAD

**Issue by junnlives** [maenDisease/Solana#6]
![6d044808-8158-4bef-8010-1b5f488dc94c](https://github.com/maenDisease/Solana/assets/69032098/16be73b9-7d69-4c29-a12c-2d83753178f1)
This is what it looks like on the loading screen, keep in mind I'm using a [config that I edited from BDEditor](https://cdn.discordapp.com/attachments/824598948257595413/1148960395269263432/Solana.theme.css)

Also having the same issue as https://github.com/maenDisease/Solana/issues/2

**Comment by maenDisease** [comment]
Are you sure the loading error is triggered by theme

**Comment by junnlives** [comment]
Yes, just tested with CSS off. Also forgot to mention I'm using Vencord

**Comment by junnlives** [comment]
Loading screen bug also looks like this after a second or so
![d5d11b84-5f45-4101-85e6-55cb47361c5b](https://github.com/maenDisease/Solana/assets/69032098/fa3df2cc-3263-4f85-84ce-c694ec6de6d8)
